src/ScnSocialAuth/Options/ModuleOptions.php

```diff
@@ -272,6 +272,16 @@ class ModuleOptions extends AbstractOptions
     protected $enableSocialRegistration = true;
 
     /**
+     * @var boolean
+     */
+    protected $debugMode = false;
+
+    /**
+     * @var boolean
+     */
+    protected $debugFile = "/temp/hybridauth.log";
```

Comment:
I will do it, copy paste fail from http://hybridauth.sourceforge.net/userguide/Debugging_and_Logging.html